projects/admin/src/app/circulation/main-request/requested-item/requested-item.component.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { RecordService } from '@rero/ng-core';
+import { Item, LoanState } from '../../../class/items';
+
+@Component({
+  selector: 'admin-requested-item',
+  templateUrl: './requested-item.component.html',
+  styleUrls: ['./requested-item.component.scss']
+})
+export class RequestedItemComponent implements OnInit {
+
+  /** requested item */
+  @Input() item: Item;
+  /** Is the detail should be collapsed */
+  @Input() isCollapsed: boolean;
+  /** the callout css class to use for this item */
+  @Input() callout: string = null;
+  /** Event emit when a request is validate */
+  @Output() requestValidated = new EventEmitter();
+
+  /** document related to the item */
+  document: null;
+
+  LoanState = LoanState;
+
+  /**
+   * Constructor
+   * @param _recordService: RecordService
+   */
+  constructor(
+    private _recordService: RecordService
+  ) { }
+
+  /**
+   * Init hook
+   */
+  ngOnInit() {
+    if (this.item) {
+      this._recordService.getRecord('documents', this.item.document.pid, 1, {
+        Accept: 'application/rero+json, application/json'
+      }).subscribe(document => this.document = document.metadata);
+    }
+  }
+
+  /** Validate a request */
+  validateRequest() {
+    this.requestValidated.emit(this.item.barcode);
+  }
+
+  /** Get the callout css code if needed */
```

Comment:
Can you be more verbose to understand the utility of this method? Ev. with a link to the official documentation.